From d695f5ed165f88b157dcc9d150a05ce756405fa7 Mon Sep 17 00:00:00 2001
From: Takuto Ikuta <tikuta@chromium.org>
Date: Thu, 25 Apr 2024 07:25:32 +0000
Subject: [PATCH] Backport "Add missing components/enterprise/buildflags
 dependency"

This is a backport of two CLs that add missing dependencies to fix a
build race condition that would sometimes lead to build errors. Both CLs
will be included in upstream's 126 release.

Upstream-Status: Backport [https://crrev.com/c/5487538, https://crrev.com/c/5526618]
Signed-off-by: Max Ihlenfeldt <max@igalia.com>
---
 chrome/browser/devtools/BUILD.gn      | 1 +
 chrome/browser/extensions/BUILD.gn    | 1 +
 chrome/browser/safe_browsing/BUILD.gn | 1 +
 3 files changed, 3 insertions(+)

diff --git a/chrome/browser/devtools/BUILD.gn b/chrome/browser/devtools/BUILD.gn
index 3f4e18b..3aa6e2f 100644
--- a/chrome/browser/devtools/BUILD.gn
+++ b/chrome/browser/devtools/BUILD.gn
@@ -115,6 +115,7 @@ static_library("devtools") {
     "//chrome/browser/autofill:autofill",
     "//components/autofill/content/browser:browser",
     "//components/autofill/core/browser:browser",
+    "//components/enterprise/buildflags",
     "//components/paint_preview/buildflags:buildflags",
     "//components/variations/service:service",
     "//components/webapps/common:common",
diff --git a/chrome/browser/extensions/BUILD.gn b/chrome/browser/extensions/BUILD.gn
index 436fef5..b284112 100644
--- a/chrome/browser/extensions/BUILD.gn
+++ b/chrome/browser/extensions/BUILD.gn
@@ -865,6 +865,7 @@ static_library("extensions") {
     "//components/embedder_support",
     "//components/embedder_support:browser_util",
     "//components/enterprise",
+    "//components/enterprise/buildflags",
     "//components/favicon/content",
     "//components/feedback",
     "//components/gcm_driver",
diff --git a/chrome/browser/safe_browsing/BUILD.gn b/chrome/browser/safe_browsing/BUILD.gn
index 9bb9b15..dce12e0 100644
--- a/chrome/browser/safe_browsing/BUILD.gn
+++ b/chrome/browser/safe_browsing/BUILD.gn
@@ -31,6 +31,7 @@ static_library("safe_browsing") {
     "//chrome/common:constants",
     "//components/browser_sync",
     "//components/enterprise:enterprise",
+    "//components/enterprise/buildflags",
     "//components/enterprise/common:strings",
     "//components/keyed_service/content",
     "//components/language/core/browser",
